### ISSUE TITLE
Can tf command run recursively?

### ISSUE THREAD

**Issue by wajdihajji** [amplify-education/terrawrap#75]

**Comment by dariuscox** [comment]
Yes, we recently added this feature with a command called graph_apply which can be found in bin/ . Preliminary documentation can be found here https://github.com/amplify-education/terrawrap/wiki/Graph-Apply, the documentation is still WIP so feel free to leave any comments, feedback, and or questions and I will try to answer the asap

**Comment by dariuscox** [comment]
Closing due to no follow-up from requester
